packages/massa-web3/src/experimental/errors/insufficientBalance.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseError } from './base'
+import { ErrorCodes } from './utils/codes'
+
+type InsufficientBalanceErrorParameters = {
+  userBalance: bigint
+  neededBalance: bigint
+  cause?: Error
+}
+
+export type InsufficientBalanceErrorType = InsufficientBalanceError & {
```

Comment:
Why do we need to export it?

Reply:
It's lighter if we only need the type which will mostly be the case for dev